src/Symfony/Component/Validator/Constraints/MacAddress.php

```diff
@@ -31,12 +42,17 @@ class MacAddress extends Constraint
 
     public function __construct(
         public string $message = 'This value is not a valid MAC address.',
+        public string $version = self::ALL,
```

Comment:
this parameter should have phpdoc documenting the valid values of the parameter (`self::ALL|self::PUBLIC|self::PRIVATE`) so that static analysis tools and IDEs can detect mistakes and provide better completion.